src/utils/logger.ts

```diff
@@ -1,68 +1,29 @@
-// import config from "config"
-import fs from "fs"
-import path from "path"
 import winston from "winston"
-import winstonDaily from "winston-daily-rotate-file"
+import Transports from "winston-transport"
 
-// logs dir
-const logDir: string = path.join(__dirname, "../logs")
-
-if (!fs.existsSync(logDir)) {
-  fs.mkdirSync(logDir)
-}
-
-// Define log format
-// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-const logFormat = winston.format.printf(({ timestamp, level, message }) => `${timestamp} ${level}: ${message}`)
-
-/*
- * Log Level
- * error: 0, warn: 1, info: 2, http: 3, verbose: 4, debug: 5, silly: 6
- */
-const logger = winston.createLogger({
-  format: winston.format.combine(
-    winston.format.timestamp({
-      format: "YYYY-MM-DD HH:mm:ss",
-    }),
-    logFormat,
-  ),
-  transports: [
-    // debug log setting
-    // eslint-disable-next-line
-    new winstonDaily({
-      level: "debug",
-      datePattern: "YYYY-MM-DD",
-      dirname: logDir + "/debug", // log file /logs/debug/*.log in save
-      filename: "%DATE%.log",
-      maxFiles: 30, // 30 Days saved
-      json: false,
-      zippedArchive: true,
-    }),
-    // error log setting
-    // eslint-disable-next-line
-    new winstonDaily({
-      level: "error",
-      datePattern: "YYYY-MM-DD",
-      dirname: logDir + "/error", // log file /logs/error/*.log in save
-      filename: "%DATE%.log",
-      maxFiles: 30, // 30 Days saved
-      handleExceptions: true,
-      json: false,
-      zippedArchive: true,
-    }),
-  ],
+const format = winston.format.printf(({ level, message, labels, timestamp, requestId }) => {
```

Comment:
Do ne need this now anymore?

Reply:
With this, logs are better formated. Eg. With this code:
```indexer_1           | 2023-08-01T07:43:16.732Z [explorer-indexer] INFO: 	indexed block on evm2: 560, domainID: 2```

without:
```indexer_1           | {"labels":{"module":"explorer-indexer"},"level":"info","message":"indexed block on evm2: 570, domainID: 2"}```

Reply:
But would this be indexed correctly on datadog then?